From b5f9685aca67cf0dded7914fa89c390891ce8210 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 17 Feb 2022 18:12:36 +0000
Subject: [PATCH] dtoverlays: Switch ov7251 to using Media Controller
 by default

We should have all the functionality required by now, so switch
to using Media Controller so that it can be used with libcamera.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/ov7251-overlay.dts | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/overlays/ov7251-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ov7251-overlay.dts
@@ -49,6 +49,7 @@
 		target = <&csi1>;
 		csi: __overlay__ {
 			status = "okay";
+			brcm,media-controller;
 
 			port {
 				csi1_ep: endpoint {
